src/parameterList.ts

```diff
@@ -39,6 +39,7 @@ export const parameterDescriptions = {
   osTmpDir: 'Override default operating system temporary directory path environment variable',
   customFlavour: 'A custom processor that can filter and process articles (see extensions/*.js)',
   optimisationCacheUrl: 'S3 url, including credentials and bucket name',
+  renderName: 'Custom render API',
```

Comment:
`forceRender: Force the usage of a specific API end-point/render, automatically chosen otherwise. Accepted values: [ VisualEditor, WikimediaDesktop. WikimediaMobile ]`. More details at https://github.com/openzim/mwoffliner/wiki/API-end%E2%80%90points`